### PR TITLE
Final updates to NI chapter

### PR DESCRIPTION
**Pull request by ChloeRN**
I have added a summary of our discussions on guidelines for using Nature Index products for Økologisk tilstand. 
I have also updated the "settings" for the example workflow to reflect these guidelines: I have dropped additional habitat indicators (and tamrein), turned off NA imputation, removed the year 1990, and changed the run type from "test" to full run (1000 iterations in bootstrap instead of 10). 

I see that the current version of the chapter code loads results from a cache. Since workflow parameters has been changed, the cache also needs to be updated.

### CODE REVIEW

**Comment by anders-kolstad** [comment]
Great. I can look at this shortly. 
Note that if you are reading data from a cache that you also include the code that creates and saves the cache. Depending on how long and essential that code is you can use echo = F or T in the code chunk options.

**Comment by ChloeRN** [comment]
I think @DrMattG added the additional chunks to load from cache when he reviewed my chapter last time...

**Review by anders-kolstad — COMMENTED** [review]
Good edits. I added some direct suggestions in a seperate PR #152 . When or if @ChloeRN accepts them she can merge it into her branch and this PR will get automatically updates as well. 

I have one question about a data file called `CustomIndex_Data.rds` which I think gets produced by one of the `NIflex` functions and stored in the root folder. Its 20MB so would be good to not have it, and I don't think it is used anywhere.

File: Nature_Index_product.Rmd [hunk 1]
```diff
@@ -335,7 +352,7 @@ CustomNI_map <- geomapNI(Index = CustomNI$CustomIndex,
 ```
 
 ```{r, echo=FALSE}
-CustomNI_map<-readRDS("data/cache/CustomNI_map.RDS")
+CustomNI_map <- readRDS("data/cache/CustomNI_map.RDS")
```
[the remark this answers is not in the document]
Comment (ChloeRN) [reply Nature_Index_product.Rmd:355]: They can be moved to wherever is convenient, I think.

File: Nature_Index_product.Rmd [hunk 2]
```diff
@@ -161,8 +161,23 @@ This is linked to methodological questions regarding how (and perhaps if) highly
...
+Area weighting is central to the method and should always be done when calculating NI products, while the need for trophic weighting depends on the indicator selection of the specific NI product: weighting by trophic group can be useful for indicator sets that are taxonomically diverse while it is largely moot for sets of indicators representing functionally and/or taxonomically closely related species. 
+For the majority of NI products for inclusion in ecosystem condition assessments key indicator weighting will not be relevant because most NI key indicators fall into the categories of either habitat indicators and/or indicators that are used independently in the assessments and are therefore removed from NI products. If any key indicators remain in the relevant indicator set after that, there is a good chance it will be only one or two, and allocating literally half the weight in a composite index to a single indicator is not reasonable. 
+
```
[the remark this answers is not in the document]
Comment (ChloeRN) [reply Nature_Index_product.Rmd:178]: Sounds good.

File: Nature_Index_product.Rmd [hunk 3]
```diff
@@ -147,7 +147,7 @@ The reference condition is defined along the same lines for NI and for other ind
 As NI products are already scaled to range from 0 to 1 and reference conditions are set according to the same approach, the reference value is 1.
 For wetlands ecosystems in particular, we have not yet discussed the definition of "good ecological condition", but it seems that in the Ecological Condition Assessments for forest and mountain ecosystems, 0.6 was used as "good ecological condition" consistent with the approach from "Vanndirektivet".
 To what degree this is appropriate is difficult to judge.
-It may well be best to use 0.6 due to its consistency with other frameworks (e.g. Vanndirektivet) as it would be challenging to otherwise determine what is "good enough" condition for a highly derived variable such as NI products.
+It may well be best to use 0.6 due to its consistency with other frameworks (e.g. Vanndirektivet) as it would be challenging to otherwise determine what is "good enough" condition for a highly derived variables such as NI products.
```
[the remark this answers is not in the document]
Comment (ChloeRN) [reply Nature_Index_product.Rmd:150]: Ah I see! I was not aware of that, thanks.

File: Nature_Index_product.Rmd [hunk 4]
```diff
@@ -321,7 +338,7 @@ CustomNI <- calculateCustomNI(ecosystem = ecosystem_use,
 ```
 
 ```{r, echo=FALSE, message=FALSE, warning=FALSE}
-CustomNI<-readRDS("data/cache/CustomNI.RDS")
+CustomNI <- readRDS("data/cache/CustomNI.RDS")
```
[the remark this answers is not in the document]
Comment (ChloeRN) [reply Nature_Index_product.Rmd:341]: Thanks!

**Comment by ChloeRN** [comment]
Completed my review of @anders-kolstad's suggestions on https://github.com/NINAnor/ecosystemCondition/pull/152. Good to go.

**Comment by ChloeRN** [comment]
@anders-kolstad I double-checked about the ``CustomIndex_Data.rds``. It does indeed not get used in the workflow here and can be deleted.

**Pull request merged**